### ISSUE TITLE
Check where data-deposition centres point for format recommendations

### ISSUE THREAD

**Issue by bansp** [clarin-eric/standards#14]
**Date stamp: 1 May 2026**

[//]: # ( $${\color{red}\text{This leading comment is being re-edited. You may want to refresh the page every now and then.}}$$ )

### 0.1. About this very note
This lead comment of ticket #14 in the repository of the [Standards Information System](https://standards.clarin.eu/sis/) (SIS) is part of the release packages of [CLARIN recommendations for data-deposition formats](https://www.clarin.eu/content/standards#clarin-recommendations-for-data-deposition-formats), and as such it gets updated at least once per release. The minimal amount of information expected to change cyclically is an update of the date string, after having confirmed that the information is current. Ideally, we are hoping for the centres listed in sections 1-4 below to gravitate towards section 5, which lists centres that maintain information in the SIS.

The information contained herein is meant to assist various bodies in the broadly conceived CLARIN governance (notably the Standards and Interoperability Committee, the BoD and the NCF, the Assessment Committee, and especially the Technical Centres Committee) in their review- and decision-making processes. 

This page is located at [`https://github.com/clarin-eric/standards/issues/14`](https://github.com/clarin-eric/standards/issues/14) . Please post remarks, updates and/or corrections in the comments section at the bottom.

### 0.2. General introduction (a.k.a. "Why bother")
CLARIN centres often offer [deposition services](https://www.clarin.eu/content/depositing-services).
[B-centres](https://www.clarin.eu/content/certified-centres) that offer such services are obligated (this is an (re-)assessment precondition, formulated as part of the [CoreTrustSeal](https://www.coretrustseal.org/) requirements) to publish explicit information about data formats that they recommend for depositions. For non-B-centres, this is not a requirement, but it is not uncommon, depending on the centre's profile and infrastructure. That obligation/practice has been encoded in one of CLARIN's Key Performance Indicators, using the following measurement: "percentage of centres offering repository services that have published an overview of formats that can be processed in their repository". (Thus the KPI measurement encompasses centres with deposition services, whereas the CTS requirement pertains to B-centres with deposition services, i.e., a subset of the KPI target group; for more details, quotes and references, see section 4 of chapter "[Standards in CLARIN](https://www.degruyter.com/document/doi/10.1515/9783110767377-012/html)", by Piotr Bański and Hanna Hedeland (2022), in the [CLARIN Book](https://www.degruyter.com/document/doi/10.1515/9783110767377/html).)

Before the SIS took wing, the requirement / good practice of publishing explicit information on recommended formats had been addressed in the following ways:
1. publishing the information somewhere at the centre (or consortium) homepage;
2. not publishing that information and instead directing users to often by now obsolete sets of recommendations (called "external guidelines" in what follows) that are far too general to represent the given centre's research profile;
3. using a mixture of the above approaches.

There was/is also a fourth group, consisting of centres with deposition services that wouldn't publish such information at all, not even as a link. It is hoped that this group is going to dissipate soon, especially thanks to the recent (2024) initiative by the Technical Centres Committee, inviting centres to deposit the relevant information in the SIS. That initiative has already borne fruit, as evidenced, among others, in the modifications of this very note between 2024 and end of 2025, whereby 11 centres have moved to section 5.

This very ticket is devoted mainly to collecting information on centres with depositing services that point to external sources of information on format recommendations, especially if those sources are not very informative. In other words, we are looking mainly at groups 2 and 3.

The reason for listing this information is: 
* to have general info on the "trends" in handling the assessment requirement, 
* to produce information potentially useful at least to the Technical Centres Committee and to the Assessment Committee,
* to identify the specific targets used by centres, in order to see if something needs/may be done about them in order to tighten the information system,
* to assist centres in publishing centre-specific data-deposition format recommendations.

Eventually, the format recommendations are expected to be collected in the [Standards Information System](https://clarin.ids-mannheim.de/standards/views/recommended-formats-with-search.xq). It is possible for centres to store that information in the SIS, and to present it to users with a dedicated link, such as:

https://standards.clarin.eu/sis/views/view-centre.xq?id=IDS

It is also possible to retrieve the information _from_ the SIS already pre-structured, as XML, to be styled according to the given centre's guidelines and publish on that centre's pages, this way avoiding the chore of maintaining two separate sets of data (for more on that, see the [API section](https://standards.clarin.eu/sis/views/api.xq) of the SIS).

### 0.3. Methodology
The primary resource assumed for this task is the CLARIN centre registry at https://centres.clarin.eu/ .

Two secondary resources are:
* the list of depositing centres at https://www.clarin.eu/content/depositing-services
* the list of B-centres at https://www.clarin.eu/content/certified-b-centres

The secondary resources appear to depend on the CLARIN centre registry and a degree of hand-crafting (and therefore a potential update lag) may probably be assumed of the depositing-services page.

A tertiary resource is the list provided by the SIS, at https://standards.clarin.eu/sis/views/list-centres.xq . While one might be tempted to assume that that list should be at least semi-automatically derived from the centre registry, it actually provides a small potential layer of indirection, at least in two aspects: firstly, we allow centres to override the shorthand handles that are listed in the registry (and thus, for example, at the centre's request, "CLARINSI" is listed as "CLARIN.SI") and, secondly, we are prepared for a degree of "ontological" or organisational variability in the case of centres that act as nodes in more than a single research-infrastructure network. In short: centres can influence their listings in the SIS in various ways, independent of the CLARIN registry.

The B-centre status is conditioned upon a successful round of certification, managed internally by the Assessment Committee, and externally by a certification authority, currently the [CTS](https://www.coretrustseal.org/) and, in the future, also nestor. (Note, incidentally, that full-fledged methodology would probably ideally start from the CTS database as the primary source, but do forgive us for not trying to shoot gnats with rockets -- the amount of time allocated to this already extensive exercise should be reasonable). The CLARIN registry has various status strings for centres that wish to achieve the B-status, whether for the first time or having lost it and preparing for another certification round -- it is, as of June 2024, "Aiming for B", "Aiming for B.", "aiming for B" (kudos for consistency) but also "none" or "Certification expired, renewal planned". In the present note, all such centres, together with "regular" C-centres, are going to be treated as "Non-B centres". Note that, especially for centres tagged as "none" in the registry, some degree of network-internal knowledge is going to be necessary for stating which of the centres are temporarily not B only because they are getting, or preparing to get, re-certified. There's no guarantee that that knowledge is perfect, so this is a weak point in the methodology.

The tables below are constructed by scrolling along the CLARIN registry and the SIS list in parallel, taking into account (a) B-centres and (b) non-B centres with deposition services that are known as such (note: this is a weak spot, some may escape, and the secondary CLARIN list is not trusted fully). In the process, the SIS list is updated wrt the CLARIN registry, and the result is sorted into the categories provided in the sections that follow. Doubts that arise wrt the nature of individual centres are usually signalled by GitHub issues that use the "[centre data](https://github.com/clarin-eric/standards/labels/centre%20data)" label.

### 0.4. Terminology
When, in what follows, a centre is said to "point to external guidelines", those guidelines are in too many cases general, top-down, coarse-grained standards recommendations that were formulated well over 10 years ago and were meant for a purpose different than informing users about centre-particular recommendations on what kinds of data the given centre can handle or is interested in handling. While such pointers are surely provided in good faith, they can at best be considered tricks for passing CTS certification. Otherwise, for practical purposes, _they don't get the thing done_.

Another relevant piece of terminology is "listed in the SIS" vs "curated": some of the content in the SIS comes from rather quick import of information that was structured rather differently back when the then Standards Committee worked with spreadsheets (that era ended around the year 2021). A lot of interpretation happened on the way between spreadsheets and the SIS, justified by the hope that the centres would quickly want to fix that if they were not happy with the outcome. It later turned out that we were a non-tiny bit too hopeful about that. The "legacy" listings, not approved by the particular centres, are accompanied by a warning in red. When, on the other hand, a centre decides to hold an [inputhon](https://github.com/clarin-eric/standards/wiki/Inputhons) and submits the result to the SIS, such recommendations are considered curated and the red warning is replaced with the name(s) of the curator(s) (see Section 5 for examples).

$${\text{* * *}}$$

What follows is information on how the particular CLARIN centres publish format recommendations or how they do _not_ publish that info while nevertheless trying to satisfy the CLARIN-internal as well as CTS-imposed requirements. Note the date stamp at the top of this note and please do not hesitate to let us know (ideally: in the comments below) if you see that some information can/should be updated or fixed.

## 1. Centres that point solely to external guidelines
This section lists centres that do not provide information specific to their research profiles but rather point to general and coarse-grained information provided by CLARIN quite a while ago, in most cases in 2009 (the "LRT Standards" document, which simply doesn't help and mentions obsolete standards).

**Note: it is good _not_ to be mentioned in this section.**

Methodological note: it is possible that the centres below also provide their own recommendations or even point at the SIS for that -- and that that mention has been overlooked (or it has been added after the date stamp in the table below).  Gathering data for this ticket has shown that some such information can be located in non-obvious places (that is very rare, but it has happened). In such cases, a question would arise as to how effective a hidden pointer to the SIS or a hidden table of recommendations is, and how that corresponds to the need of satisfying a KPI or a CTS/assessment requirement.

### 1.1. B-centres
Recall that B-centres are obligated (by CTS Requirement 8) to provide explicit information on what formats they are willing to receive in the deposition process. The centres below instead point to general and at least partly obsolete guidelines. Amending this situation is at this point easy: [deposit the relevant information directly in the SIS](https://github.com/clarin-eric/standards/wiki/Updating-format-recommendations) -- and then point to that description -- as the centres listed in section 5 below have already done.

| Centre | LastChecked | LinkTarget | SourcePage |
| --- | --- |--- | --- |
| [ERCC](https://clarin.ids-mannheim.de/standards/views/recommended-formats-with-search.xq?centre=ERCC) | 29-10-2025 | http://www.clarin.eu/sites/default/files/Standards%20for%20LRT-v6.pdf | https://clarin.eurac.edu/repository/xmlui/page/faq#what-submissions-do-you-accept |
| [ILC4CLARIN](https://standards.clarin.eu/sis/views/view-centre.xq?id=ILC4CLARIN) | 01-05-2026 | http://www.clarin.eu/sites/default/files/Standards%20for%20LRT-v6.pdf | https://dspace-clarin-it.ilc.cnr.it/faq#what-submissions-do-you-accept |
 
* As of nearly mid-December, ILC4CLARIN and LINDAT have switched to d7. The unhelpful link remained in ILC but LINDAT, as of at least April 2026, is pointing at the SIS.

### 1.2. Non-B centres
These centres are not obligated to explicitly publish information about what formats they recommend for deposition. However, that is both useful for the users themselves, and also crucial for satisfying the relevant CLARIN KPI. Also, these centres are [listed as aiming for the "B" status](https://centres.clarin.eu/) (click on "Type status" to sort them at the top), so at some point they will need to undergo CTS assessment -- why not be proactive in this respect.

| Centre | LastChecked | LinkTarget | SourcePage |
| --- | --- |--- | --- |
| [CLARIN-LT](https://standards.clarin.eu/sis/views/view-centre.xq?id=CLARIN-LT) | 01-05-2026 | http://www.clarin.eu/sites/default/files/Standards%20for%20LRT-v6.pdf | https://clarin-repo.lt/info/faq#q2 |
| [SADiLaR](https://standards.clarin.eu/sis/views/view-centre.xq?id=SADiLaR) | 11-12-2025 | http://www.clarin.eu/recommendations <br /> https://archive.mpi.nl/accepted-file-formats | https://sadilar.org/en/submit-a-resource/ |

* SADiLaR points to mpi.nl recommendations, which is definitely more helpful than just pointing to the "LRT standards" document, although a question arises as to whether the recommendations pointed to are actually SADiLaR's own recommendations, given that SADiLaR doesn't have any control over the referenced list. But that is for the CTS to assess. Pointing to the "recommendations" page at clarin.eu is a point in favour.
  * It is also worth mentioning that, in April 2025, [MPI-PL](https://standards.clarin.eu/sis/views/view-centre.xq?id=MPI-PL) deposited their recommendations in the SIS. In other words, a template already exists in the SIS, so perhaps not a lot of work would have to be devoted to making the list SADiLaR's own.
* As of April 2026, the LT link has changed, but the new FAQ still points to the obsolete LRT-v6 document.

## 2. Centres that point to external guidelines in addition to publishing own information locally
Just like centres listed in section 4 below, those listed in this section fulfil the CTS requirements by publishing explicit recommendations concerning formats in which data can be deposited with them. **From the formal point of view, they have done a splendid job.** The place where their own recommendations are published are listed in the last column of the table below.

The focus of this note is to see where centres point for external information, and, in particular, to catalogue the (let's call them) suboptimal places where users are directed, so that something can be done about that. There is nothing wrong in pointing to an external source in addition to the centre's own recommendations published on the centre's own pages, especially if the external resource brings in some extra value (see ACDH-ARCHE for an example, pointing to Archeology Data Service recommendations). On the other hand, it is not so good to point to obsolete, unhelpful or misleading documents, such as the "LRT standards" PDF.

Thus, the role of this section is basically informative, though with a request directed at the centres enumerated below, to consider sharing at least the positive recommendations in the SIS, in order to enable aggregation of this information and publishing it for the benefit of the community. 

| Centre | LastChecked | LinkTarget | SourcePage | Own info |
| --- | --- |--- | --- |--- |
| [ORTOLANG](https://standards.clarin.eu/sis/views/view-centre.xq?id=ORTOLANG) | 01-11-2025 | https://www.clarin.eu/content/standard-recommendations | https://www.ortolang.fr/en/help/data-formats/ <br /> https://www.ortolang.fr/en/help/long-term-archiving/ | https://facile.cines.fr/ |

* For ORTOLANG, the user have to go to `Help | Publishing | Long-term archiving` in order to reach the entry to the recommendations, which are located at Cines, which also has an interesting info page at https://www.cines.fr/preservation/expertises/file-format/conversions/
* BBAW left this section in October 2025.

## 3. Centres that neither point anywhere nor publish their own explicit information
This set of centres should be empty, _unless the centre does not offer deposition services_ (in which case, it shouldn't be listed here, so... this set should be empty). Please note that rather than amending the existing lack of recommendations on their own home pages, the best course of action for these centres may be to [deposit the information directly in the SIS](https://github.com/clarin-eric/standards/wiki/Updating-format-recommendations), and then point to that listing. You just do one [inputhon](https://github.com/clarin-eric/standards/wiki/Inputhons) and Bob's your... list.

### 3.1. B-centres
"Absence of evidence is not evidence of absence", and it might be that the centres here do publish their own recommendations, in a non-obvious corner of their homepages. Please feel very welcome to post a comment below if you are able to share info on that. Note also that if the info is hidden then it's not really easily available to the depositing users, and ensuring availability of the information is part of the reason for this entire exercise.

| Centre | LastChecked | Comment | DepositionPage |
| --- | --- |--- | --- |
| [HR-CLARIN](https://standards.clarin.eu/sis/views/view-centre.xq?id=HR-CLARIN) | 11-12-2025 | no recommendations, no login | https://repository.clarin.hr/home |
| [MI](https://standards.clarin.eu/sis/views/view-centre.xq?id=MI) | 11-12-2025 | "please contact [us]" | https://meertens.knaw.nl/meertens-collectie/research-data-management/ <br /> https://meertens.knaw.nl/en/archive/depositing-data_eng_/ |

* Note that HR is a a very fresh B-centre and its position on this page will hopefully change soon. The repository is at https://repository.clarin.hr/home and it might be possible to see some recommendations once you're logged in, but Shibboleth fails (Oct 30th; management notified).
* MI just says "If you consider storing your data at the Meertens Institute or have any questions, please contact Douwe Zeldenrust, Collection Manager, for information about the procedure." That is not exactly the solution required by the CTS.
* MI: the page in Dutch provides more overall information, but neither language version provides a list of formats recommended by the centre

### 3.2. Non-B centres
Some of these centres are listed as "aiming for B", some used to be B. All of them indicate that they provide deposition services. 

| Centre | LastChecked | Comment | DepositionPage |
| --- | --- |--- | --- |
| [CELR-EKK](https://standards.clarin.eu/sis/views/view-centre.xq?id=CELR-EKK) | 31-10-2025 | "all data is accepted", via Entu | https://www.keeleressursid.ee/en/services |

* CELR-EKK's is a META-SHARE repository, Shibboleth login is not possible (to check whether there is a list of recommended formats inside the registered space), the help file does not mention deposition or formats.
* IMS moved from here to Section 3.3 in October 2025.

### 3.3. Special mention: the departed
An argument could be forced that former repositories qualify in this section because their neither point nor publish -- true enough. Apart from that, I didn't want to change section numbering in order to add such a section, while, on the other hand, it seemed sensible not to let these centres vanish without a trace. They will probably be omitted in the next recommendations release, unless their deposition services are restored.

**3.3.1.** The C-centre [CEDIFOR](https://centres.clarin.eu/centre/35) used to "point elsewhere" for data deposition recommendations at the beginning of the history of this ticket. (There is a history of extremely unclear information trickle from the maintainer, since June 2024, which I suppress in the final 2025 report.)

The [last version with a link to the repository](https://web.archive.org/web/20211213032438/https://www.cedifor.de/) was captured in December 2021. The address is https://www.cedifor.de/  (back online only since 06Nov25, with thanks to Dieter; no information about repository services, so the centre stays in this category until the next iteration of checks, when the mention of it will most probably be removed altogether, unless deposition services come back).

**3.3.2.** [CMU](https://standards.clarin.eu/sis/views/view-centre.xq?id=CMU) appears to have closed its doors on deposition in 2025 -- the [last snapshot of the deposition page](https://web.archive.org/web/20250412135216/https://talkbank.org/share/contrib.html) is from April. The distribution and information page remains https://talkbank.org/ .

**3.3.3.** [IMS](https://standards.clarin.eu/sis/views/view-centre.xq?id=IMS) still has the info page at  https://wiki.ims.uni-stuttgart.de/extern/CLARIN-D , and the single question there "Is the data in one of the acceptable formats (non-proprietary, text-based) or can it be converted?" classified the centre in Section 3.
* The IMS "repo" page (http://clarin04.ims.uni-stuttgart.de/repo/) was not accessible on 05-10-2024, and neither was it there in October 2025. If one looks at the (old) [CTS certification document (PDF)](https://www.coretrustseal.org/wp-content/uploads/2018/12/IMS-Repository.pdf) for IMS then it looks like that "repo" link is the link to the repository, so... it looks like the repository is gone, in which case the issue of whether recommendations exist is rather moot. Breaking news: the [last snapshot of working repository](https://web.archive.org/web/20230419183945/http://clarin04.ims.uni-stuttgart.de/repo/) was taken in April 2023.
* It needs to be stressed that the resources deposited with IMS are still accessible at https://www.ims.uni-stuttgart.de/forschung/ressourcen/

## 4. Centres that only publish their own, local recommendations
Note that this satisfies both the CTS requirement and the KPI calculation (though not the [KPI calculation performed dynamically by the SIS](https://standards.clarin.eu/sis/views/kpi.xq)). Unfortunately, it also ensures a gap in the SIS-derived statistics that might otherwise benefit the entire network. It would be greatly appreciated if at least the data formats _recommended_ by the centres could make it into the SIS. The table below includes both B- and C-centres. 

Note: Formally, these centres at least satisfy the CTS requirements, and in some cases provide a very comprehensive list of recommendations. Adding their recommendations to the SIS would be a serious step towards ensuring more accurate statistics.

| Centre | LastChecked | Linked from / Comment | InfoPage |
| --- | --- |--- | --- |
| [BAS](https://standards.clarin.eu/sis/views/view-centre.xq?id=BAS) | 31-10-2025 | https://www.bas.uni-muenchen.de/forschung/Bas/BasInfoStandardsTemplateseng.html | https://www.bas.uni-muenchen.de/forschung/Bas/BasFormatseng.html | 
| [BBAW](https://standards.clarin.eu/sis/views/view-centre.xq?id=BBAW)  |29-10-2025 | https://clarin.bbaw.de/en/ | https://clarin.bbaw.de/en/repo/ |
| [CLARIN-DK](https://standards.clarin.eu/sis/views/view-centre.xq?id=CLARIN-DK-UCPH) | 12-12-2025 | https://repository.clarin.dk/repository/xmlui/page/faq#what-data-formats-are-accepted | https://repository.clarin.dk/repository/xmlui/page/formats | 
| [COCOON](https://standards.clarin.eu/sis/views/view-centre.xq?id=COCOON) | 01-11-2025 | https://cocoon.huma-num.fr/exist/crdo/faq.htm?lang=en | https://cocoon.huma-num.fr/exist/crdo/formats.htm?lang=en |
| [DANS](https://standards.clarin.eu/sis/views/view-centre.xq?id=DANS) | 01-11-2025 | https://dans.knaw.nl/en/depositing-data-manual/before-depositing_ds/ | https://dans.knaw.nl/en/file-formats/ |
| [DH-Graz](https://standards.clarin.eu/sis/views/view-centre.xq?id=DH-Graz) | 01-11-2025 | https://digital-humanities.uni-graz.at/de/ | https://gams.uni-graz.at/context:gams?mode=about&locale=en |
| [EKUT](https://standards.clarin.eu/sis/views/view-centre.xq?id=EKUT) | 01-11-2025 | https://talar.sfs.uni-tuebingen.de/resources/ | https://talar.sfs.uni-tuebingen.de/datamanagement/ |
| [LAC](https://standards.clarin.eu/sis/views/view-centre.xq?id=LAC) | 01-11-2025 | https://dch.phil-fak.uni-koeln.de/bestaende/language-archive-cologne/user-guides | https://dch.phil-fak.uni-koeln.de/bestaende/language-archive-cologne/user-guides/format-whitelist |

* BBAW moved here from Section 2 in October 2025. The CLARIN-D guide it used to point the user at 
(https://www.clarin-d.net/en/language-resources-and-services/user-guide ) is a dead link (as of October 2025). Effectively, only a brief, prose list of formats remains.
* Given their exceptionally rich recommendations and the fact that they serve many areas of research, it is reasonable to expect DANS to remain as it is. Ergo: 100% of curated centres is not likely, unless mirroring DANS in the SIS becomes an intern-level cyclic task. Big question: should we comment out the existing recommendations in the SIS that have not been authorised by DANS, or keep them as a seed for the aforementioned hypothetical intern task?
* DH-Graz (formerly ZIM) uses GAMS as the repository; it is a repository for digital data from the Humanities / SocSci. Getting from DH-Graz to GAMS requires the user to get to the menu item "unsere Forschung / our research" and selecting "Research data management"  -- fair enough. It feels like the "About" page of GAMS is much more comprehensive than when I last checked it. They also have an interesting further page https://gams.uni-graz.at/doku
* EKUT = TALAR, MPI-PL = TLA
* EKUT's former page with format recommendations was https://talar.sfb833.uni-tuebingen.de/datamanagement/ -- it is probably dead. A working link is https://fdat.uni-tuebingen.de/communities/talar from https://talar.sfs.uni-tuebingen.de/resources/ .

## 5. Centres that point at their curated recommendations in the SIS
This is where all (or most of) the centres listed above should ideally end up -- what is needed for them is to maintain the information served by the SIS and _explicitly link to it_.

| Centre | LastChecked | SourcePage | 
| --- | --- |--- |
| [ACDH-ARCHE](https://standards.clarin.eu/sis/views/view-centre.xq?id=ACDH-ARCHE) | 12-12-2025 | https://arche.acdh.oeaw.ac.at/browser/filenames-formats-metadata |
| [CLARIN-CH](https://standards.clarin.eu/sis/views/view-centre.xq?id=CLARIN-CH) |  01-11-2025 | https://clarin-ch.ch/documentation-platform/standard-data-formats |
| [CLARIN:EL](https://standards.clarin.eu/sis/views/view-centre.xq?id=CLARIN-EL) | 02-11-2025 | https://www.clarin.gr/en/services/share |
| [CLARIN-IS](https://standards.clarin.eu/sis/views/view-centre.xq?id=CLARIN-IS) | 02-11-2025 | https://repository.clarin.is/repository/xmlui/page/submission-guidelines |
| [CLARIN-LV](https://standards.clarin.eu/sis/views/view-centre.xq?id=CLARIN-LV) | 06-01-2026 |   https://repository.clarin.lv/repository/xmlui/page/faq |
| [CLARIN-PL1](https://standards.clarin.eu/sis/views/view-centre.xq?id=CLARIN-PL1) | 01-11-2025 | https://clarin-pl.eu/dspace/page/data |
| [CLARIN.SI](https://standards.clarin.eu/sis/views/view-centre.xq?id=CLARIN.SI) | 02-11-2025 |  https://www.clarin.si/repository/xmlui/page/data |  
| [CLARINO_Bergen](https://standards.clarin.eu/sis/views/view-centre.xq?id=CLARINO_Bergen) |  01-11-2025 | https://repo.clarino.uib.no/xmlui/page/faq#what-submissions-do-you-accept |
| [FIN-CLARIN](https://standards.clarin.eu/sis/views/view-centre.xq?id=FIN-CLARIN) | 01-11-2025  | https://www.kielipankki.fi/tuki/tekninen-muoto/ |
| [IDS](https://standards.clarin.eu/sis/views/view-centre.xq?id=IDS) | 01-11-2025 | https://repos.ids-mannheim.de/reposdescription.html |
| [INT](https://standards.clarin.eu/sis/views/view-centre.xq?id=INT) | 02-11-2025 | https://portal.clarin.ivdnt.org/information-about-deposition.html | 
| [OTA](https://standards.clarin.eu/sis/views/view-centre.xq?id=OTA) |02-11-2025 | https://llds.ling-phil.ox.ac.uk/llds/xmlui/page/faq#what-submissions-do-you-accept |
| [PORTULAN](https://standards.clarin.eu/sis/views/view-centre.xq?id=PORTULAN-CLARIN) | 02-11-2025  | https://portulanclarin.net/usage/#how |
| [SAW](https://standards.clarin.eu/sis/views/view-centre.xq?id=SAW) | 02-11-2025 | https://repo.data.saw-leipzig.de/depositing/en |
| [Språkbanken](https://standards.clarin.eu/sis/views/view-centre.xq?id=Sprakbanken) | 02-11-2025 | https://repo.spraakbanken.gu.se/xmlui/page/faq#what-submissions-do-you-accept |
| [TROLLing](https://standards.clarin.eu/sis/views/view-centre.xq?id=TROLLing) | 05-11-2025 |  https://site.uit.no/dataverseno/deposit/prepare/#what-are-preferred-file-formats |
| [UdS](https://standards.clarin.eu/sis/views/view-centre.xq?id=UdS) | 02-11-2025 | https://fedora.clarin-d.uni-saarland.de/depositors.en.html |

* CLARIN-CH does not provide a direct link to the centre's page in the SIS, but it does provide a link to the aggregated recommendations
* CLARIN-IS references the entire SIS (no direct link to the centre's SIS page), and then really nicely references the particular format pages in the SIS, except that at least one link doesn't work (Wave), and at least one link leads to a format (PDF/A), on whose page CLARIN-IS is _not_ listed as a centre that recommends it. So the whole thing might benefit from a review. 
* ARCHE points additionally at https://archaeologydataservice.ac.uk/help-guidance/instructions-for-depositors/files-and-metadata/ (which, to be sure, is a bonus)

### Waiting room
These centres do not appear to link back to the SIS, although their recommendations are placed here. That would not be a very economical move, so a likely interpretation is that it was just an oversight that can be easily fixed.

| Centre | LastChecked | SourcePage | 
| --- | --- |--- |
| [DH-rep](https://standards.clarin.eu/sis/views/view-centre.xq?id=DH-rep) | 05-11-2025 | https://repository.de.dariah.eu/doc/services/data-policies.html#recommendations-and-list-of-preferred-formats | 
| [MPI-PL](https://standards.clarin.eu/sis/views/view-centre.xq?id=MPI-PL) | 05-11-2025 |  https://archive.mpi.nl/tla/accepted-file-formats |
| [TG-rep](https://standards.clarin.eu/sis/views/view-centre.xq?id=TG-rep) | 05-11-2025 | https://textgridlab.org/doc/services/data-policies.html#preferredformats | 
| [HZSK](https://standards.clarin.eu/sis/views/view-centre.xq?id=HZSK) | 05-11-2025 | https://www.fdr.uni-hamburg.de/communities/hzsk/ <br /> https://www.fdr.uni-hamburg.de/| 

## 6. Point at the SIS but do not store info there
This category was created in May 2026 especially for LINDAT :-)

| Centre | LastChecked | LinkTarget | SourcePage |
| --- | --- |--- | --- |
| [LINDAT](https://standards.clarin.eu/sis/views/view-centre.xq?id=LINDAT) | 01-05-2026 | https://standards.clarin.eu/sis/views/recommended-formats-with-search.xq | https://lindat.cz/faq-repository#what-submissions-do-you-accept |

It should be considered a kind of a waiting room, like the one in section 5 above, except, in this case, there are no curated recommendations in the SIS, and the repository points at the SIS list of recommendations -- which is _way_ better than pointing at the LRT-v6 document, so, overall, it's a win! :-)

## 7. Conclusions
7.1. One conclusion that should be drawn from the picture above that the FAQ contained in the LINDAT customisation of DSpace (the deposition system that unifies many repositories, currently) should no longer point users at the LRT PDF but:
* by default, reference e.g. https://www.clarin.eu/content/standard-recommendations
* but also encourage the individual centres to customise the link and point directly at the centre's listing in the SIS.

As of October 2024, this is being handled in issue [1124 at UFAL's repository](https://github.com/ufal/clarin-dspace/issues/1124).

7.2. Following up on the above, the default landing page (`content/standard-recommendations`) should, at the top, point at the [combined recommendations in the SIS](https://standards.clarin.eu/sis/views/recommended-formats-with-search.xq). (that got handled on 17-06-2024)

7.3. Centres which provide their own extensive recommendations (cf. section 4) will hopefully be willing to share at least their recommended (as opposed to accepted and discouraged) formats, so that (a) the KPI can be properly calculated in the SIS, and (b) so that the statistics of [popular formats](https://standards.clarin.eu/sis/views/list-popular-formats.xq?threshold=1&top3=Most+recommended+formats) are not skewed due to the lack of data coming from those centres. This can be a matter of argumentation and appeal to the "common good" vs. the various restrictions on those centres (time being the most commonly cited one).

7.4. It seems reasonable not to expect DANS to convert to the SIS, but a slim chance exists that DANS can mirror their recommendations at the SIS. That would probably require an intern-level task, with a plan for cyclic curation.

7.5. (As of November 2025:) For the three (3!) centres in section 3.3, CEDIFOR, CMU and IMS, it probably makes sense to remove the "deposition" flag, together with their recommendations (which were never curated). Whether they then remain in the SIS or not is not critically relevant. [handled in commit [54e6814](https://github.com/clarin-eric/standards/commit/54e681405c1134cf51d5f8ddf03e403807edc0c9)].

7.6. In some cases (one I can recall at the moment of writing is TROLLing), the local recommendations appear to be a superset of the recommendations shared with the SIS. One reason that comes to mind is that, perhaps, it is not sufficiently strongly stressed that the SIS can accommodate recommendations pointing at formats not yet described here. Another thing that needs to be stressed is that the functional domain system is going to adjust to the needs of centres and, indirectly, to the inventory of formats referenced by recommendations: the fact that some domain is not yet named may simply mean that we need feedback or, sometimes, assistance in adding it.

**Comment by bansp** [comment]
Let us edit the leading note, to extend (or shrink!) the table.
The table might be a good addendum to the April release, so let me set a milestone here.

**Comment by TomazErjavec** [comment]
I agree, interesting table, thanks!

**Comment by bansp** [comment]
I seem to recall Leif-Jöran pointing me to a stock data deposition guidelines for the Sprakbanken (pointing at the "Standards for LRT" PDF, I think), but I am unable to locate that page now at https://spraakbanken.gu.se/en

**Comment by bansp** [comment]
> I seem to recall Leif-Jöran pointing me to a stock data deposition guidelines for the Sprakbanken (pointing at the "Standards for LRT" PDF, I think), but I am unable to locate that page now at https://spraakbanken.gu.se/en

Thanks to Hanna for digging up https://repo.spraakbanken.gu.se/xmlui/page/faq#what-submissions-do-you-accept for me. I spent quite a while at the Sprakbanken site yesterday night, trying to find my way to the deposition guidelines as a "naive user coming from outside", which makes me wonder how (un)easy it is to find that. I'd be grateful for an independent check.
And in the meantime, I'll update the table.

**Comment by bansp** [comment]
OK, the way to the deposition info at Gothenburg is through the "Tools" in the menu, then one has to navigate to the item mentioning CLARIN, and that takes them to the repository page. So I simply overlooked this route yesterday.

**Comment by bansp** [comment]
I've just gone through all the links and verified that the info is current. Getting the ticket exported as PDF requires a lot of tinkering in the "Inspect" box and then using custom zoom (55%) in the print window.

**Comment by bansp** [comment]
Update needed to mention Iceland:
https://repository.clarin.is/repository/xmlui/page/faq#what-submissions-do-you-accept 

+ divide the list into three rather than two, with a separate part for centres that do mention their own recommendations while also referencing the "LRT standards" document (mostly that one, because of the stock FAQ).

**Comment by bansp** [comment]
Constanza Navarretta has just kindly pointed me to the recommendations for CLARIN-DK:
https://info.clarin.dk/en/the-clarin-dk-infrastructure/recommended-standards-and-formats/

I'll redo the table in 3 parts within daaays, I hope.

**Comment by bansp** [comment]
(Ah, the reason CLARIN-DK isn't mentioned above is that it doesn't point externally, and the topic of this ticket is external pointers)

**Comment by bansp** [comment]
While updating the info, I'm unable to access https://repo.spraakbanken.gu.se/xmlui -- making a note of that here, to check again on Monday.

**Comment by bansp** [comment]
I will take a snapshot of the ticket on Monday afternoon, publish the snapshot and reset the milestone to 1.1.

**Comment by bansp** [comment]
Posted a [snapshot](https://www.clarin.eu/content/standards#release-1-0-november-2021), moving the ticket to milestone 1.1.

**Comment by bansp** [comment]
The DSpace FAQ is here, I believe:
https://github.com/ufal/clarin-dspace/blob/clarin/dspace-xmlui/src/main/webapp/themes/UFAL/lib/html/faq.html

**Comment by bansp** [comment]
I have gone through the tables above and posted short updates. In a few cases, I crossed a centre out (that's actually good!), in one case, I moved a centre from "aiming at B" to "B" (congrats!). This was a quick check, so in case you seen an error or omission, please post a note here.

Overall impression after 2 years: revolution still needs to happen.

**Comment by bansp** [comment]
Maria Gavriilidou has sent me the following info on CLARIN:EL:

> - the link to the file with all the accepted & recommended formats is this https://www.clarin.gr/sites/default/files/CLARINELRecommendedFormats.pdf
> - which can be found here https://www.clarin.gr/en/services/share, together with the Data Collection Policy and Preservation Policy documents. We plan to change their position in the portal, so that they can be more easily found, but for the time being this is their location.

**Comment by bansp** [comment]
For the record, this is the content of my comments in the CLARIN progress report document, concerning the KPI calculation. The initial motivation of my comments was that, apparently, only B-centres were counted for the relevant KPI, whereas it seems that that is not consistent with what the description of the KPI says.

"Centres offering depositing services" are not only B-centres. I count currently 42:
* numbers: https://standards.clarin.eu/sis/views/list-statistics-centre.xq
* names: https://standards.clarin.eu/sis/views/list-centres.xq
Currently, I count (but this calculation may be imprecise wrt CLARIN registry, we sync only occasionally because we don't know when changes in the registry occur) 22 B-centres: https://standards.clarin.eu/sis/views/list-centres.xq?status=B-centre&submit=Filter
(but: see above, these are just B-centres, and the KPI does not say "B-centres"). Note that only 4 out of these 22 have submitted their preferences to the SIS. Some of them, however, publish their preferences on their own. 

A different set of calculations that I last updated on 23 July 2024 comes from https://github.com/clarin-eric/standards/issues/14 (i.e., this ticket).
Note that this ticket does not fully distinguish between B-centres and others, because it focuses on centres with depositing services, mentioning B-centres only in cases where they either seem not to fulfil certification criteria or when they skirt them (mostly in good faith, probably) by referencing a by now obsolete document in the clarin.eu domain that actually says something different than those centres are expected to say.
So, this ticket has 42 as its basis. Out of the 42, I count:
* 8 centres that publish their own information and point somewhere else (sometimes sensibly, sometimes not) -- see section 2
* 12 centres that publish this information on their pages only (section 4)
* 6 centres that publish their preferences via the SIS (section 5)
Apart from that:
* 7 centres that link somewhere instead of actually providing the relevant information (section 1)
* 1 centre (Språkbanken) with an unclear status in this respect: recommendations in the SIS but not pointing to them from their own page (instead pointing elsewhere)

Based on the above, the most restrictive count (and including Språkbanken, sigh) is: 8+12+6+1 = 27 against 42 (~ 64%). Including centres that point at the sky saying that they fulfil the KPI this way (7), we get 34 against 42 (~ 81%).

**Comment by bansp** [comment]
Note: a lot has happened since Sep 28th (the date of the last comment) and October 23rd (the current date stamp). EL is now curated as part of the SIS, and IVDNT and ARCHE, and .SI are preparing to place their recommendations in the SIS.
A new recommendations release is going to be published some time in the first half of 2025, hopefully with at least the three above-mentioned centres on board.

**Comment by TomazErjavec** [comment]
@bansp, what about copying the intro part of this issue into the main README file, which is now quite laconic.
And it can be further edited there, where everybody can see it. Alternativelly, into /doc as GitHub page?
Just an idea...

**Comment by bansp** [comment]
I have just moved 11 (!) centres to Section 5. Apologies for taking so long with some of them. The ticket is still undergoing changes: I am going to re-traverse the links, update date stamps, change some more comments, etc.. 
Hey, what a really nice feeling this is!
